src/components/Error/Error.tsx

```diff
@@ -81,30 +68,44 @@ export const Error = ({
     error && getErrorInfo(error).codes.includes('NotFound') ? 404 : 500;
 
   return (
-    <Component className={cx(page && classes.page)}>
-      {/* Default status code to be helpful for the most common ones. The
+    <Box
+      component={Component}
+      sx={(theme) => ({
+        overflow: 'auto',
+        padding: theme.spacing(4, 0, 0, 4),
+      })}
```

Comment:
Re-introduce this style behind the page flag.
```suggestion
      sx={[!!page && { overflow: 'auto', pt: 4, pl: 4, }]}
```